### DISCUSSION TITLE
BBC Micro Emulator on Miyoo Mini Plus?

### DISCUSSION THREAD

**Discussion by bteamfox** [OnionUI/Onion#1312] [General]
Hi, 
Does anyone know if there is a BBC Micro emulator for this device? I was surprised there is no Retroarch Core as The BBC Micro was a Hugh device in the UK. Almost all schools back in the 80’s had at least one and was a popular home computer. Elite was and important game on the BBC. I love the Miyoo Mini Plus it just saddens me I can’t find a way to run BBC Micro games. Elite, Revs, Chuckie Egg, Starship Commander, Mr. Ee! That’s just a few games I would love to play.

Thank you.

**Comment by schmurtzm** [comment]
Hi, I think that AdvanceMess is supporting BBC Micro, AdvanceMame/AdvanceMess are already available in Package Manager under the expert section.
However AdvanceMess is not easy at all to configure, you'll have to document yourself on how to configure it. However even if no one use it this AdvanceMess is working in Onion, I have already tested it (and it support a LOT of systems !).
When you install AdvanceMame you'll have some game samples, I recommend to try to make it work first.

You can read [the documentation I've made about it here](https://github.com/OnionUI/Onion/blob/main/static/packages/App/AdvanceMENU%20(Alternative%20frontend)/App/AdvanceMENU/info.txt).
